vim/yanking-text-to-register.md

```diff
@@ -0,0 +1,21 @@
+# Yanking text to register
+
+Yanking and deleting text will save the text to the same unnamed register. Which means every time you delete a word, your 
```

Comment:
How about replacing "Which means.." with  "This means..." or even just "Every time you delete..." ?